### PR TITLE
Add initial logging to JHU indicator

### PR DESCRIPTION
**Pull request by benjaminysmith**
### Description
Add initial (trivial) logging to JHU indicator so that we start collecting log data

### Changelog
- Adds simple logging to main JHU code.

### Fixes 
- Partial fix to #606

### CODE REVIEW

File: jhu/delphi_jhu/run.py [hunk 1]
```diff
@@ -86,6 +87,12 @@ def run_module():
         METRICS, GEO_RESOLUTIONS, SENSORS, SMOOTHERS
     ):
         print(metric, geo_res, sensor, smoother)
+        logger.info(
+            event="processing data",
```
Comment (chinandrew) [inline jhu/delphi_jhu/run.py:91]: is this descriptive enough (e.g. isn't everything in this pipeline "processing data")? I guess the logs will have file/indicator name and the metric/geo/sensor/smoother which seems sufficient, but haven't followed the logging so don't know how these all get used/parsed.

Comment (benjaminysmith) [reply jhu/delphi_jhu/run.py]: I updated to be a bit more specific here, but yeah we should have lots of information to use in the logs. We'll likely go back and edit these as we start understanding what it most helpful to collect.

File: jhu/delphi_jhu/run.py [hunk 2]
```diff
@@ -86,6 +87,12 @@ def run_module():
         METRICS, GEO_RESOLUTIONS, SENSORS, SMOOTHERS
     ):
         print(metric, geo_res, sensor, smoother)
+        logger.info(
+            event="exporting to CSV",
```
Comment (chinandrew) [inline jhu/delphi_jhu/run.py:91]:
```suggestion
            event="generating signal and exporting to CSV",
```

**Review by chinandrew — APPROVED** [review]
one suggestion, otherwise :shipit: :shipit: :shipit:

**Pull request merged**